### PR TITLE
Installer tests

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by jtydhr88** [comment]
ft retest

**Comment by jtydhr88** [comment]
it retest

**Comment by jtydhr88** [comment]
it retest

File: build/installers/installer-tests/pom.xml [hunk 1]
```diff
@@ -1,56 +1,59 @@
...
-        </plugin>
-    </plugins>
-</build>
-</project>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>5.1.0</version>
+
```
Comment (gamerson) [inline build/installers/installer-tests/pom.xml]: Remove extra new lines

File: build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/DevStudioCETest.java [hunk 2]
```diff
@@ -14,50 +14,70 @@
 
 package com.liferay.ide.installer.tests;
 
+import com.liferay.ide.installer.tests.checker.app.AppChecker;
```
Comment (gamerson) [inline build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/DevStudioCETest.java]: This should not be in `src/test/java` but in `src`

File: build/installers/installer-tests/pom.xml [hunk 3]
```diff
@@ -1,56 +1,59 @@
...
+	</dependencies>
+	<build>
+		<plugins>
+
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.19.1</version>
+				<dependencies>
+					<dependency>
+						<groupId>org.junit.platform</groupId>
```
Comment (gamerson) [inline build/installers/installer-tests/pom.xml]: sort dependencies

File: build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/checker/app/AppCheckerFactory.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.installer.tests.checker.app;
+
+/**
+ * @author Terry Jia
+ */
+public class AppCheckerFactory {
```
Comment (gamerson) [inline build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/checker/app/AppCheckerFactory.java]: Lets remove this class.

File: build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/checker/file/FileCheckerFactory.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.installer.tests.checker.file;
+
+/**
+ * @author Terry Jia
+ */
+public class FileCheckerFactory {
```
Comment (gamerson) [inline build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/checker/file/FileCheckerFactory.java]: Lets remove this class.

File: build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/checker/process/ProcessFactory.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+ * details.
+ */
+
+package com.liferay.ide.installer.tests.checker.process;
+
+/**
+ * @author Ashley Yuan
+ * @author Terry Jia
+ */
+public class ProcessFactory {
```
Comment (gamerson) [inline build/installers/installer-tests/src/test/java/com/liferay/ide/installer/tests/checker/process/ProcessFactory.java]: Lets remove this class.

File: build/installers/installer-tests/pom.xml [hunk 7]
```diff
@@ -1,56 +1,59 @@
...
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<version>5.1.0</version>
+
+		</dependency>
+		<dependency>
+			<groupId>org.junit.vintage</groupId>
+			<artifactId>junit-vintage-engine</artifactId>
```
Comment (gamerson) [inline build/installers/installer-tests/pom.xml]: If we are writing tests using junit jupitor api but we have the vintage engine that means our tests will still be junit4 right?  Do we plan to update these tests using junit 5/jupitor features?

**Pull request closed without merging**